scripts/test-with-kind.sh

```diff
@@ -63,7 +63,7 @@ function install_controller {
        local __controller_name="appmesh-controller"
        local __ns="appmesh-system"
        kubectl create ns $__ns
-       APPMESH_PREVIEW=y AWS_ACCOUNT=$AWS_ACCOUNT_ID AWS_REGION=$AWS_REGION make helm-deploy
+       APPMESH_PREVIEW=y AWS_ACCOUNT=$AWS_ACCOUNT_ID AWS_REGION=$AWS_REGION ENABLE_BACKEND_GROUPS=true make helm-deploy
```

Comment:
this will enable the backend groups for all tests (which doesnt sound bad tbh) but if you wanted to gate this for only your integration test suite you can do a separate helm-deploy as a case on L80 like i do in my PR here https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/598/files#diff-af7f07af62d09eec6acc7e70d0f073dcd4b6326b52e02853b2adc6a8984fe73eR90